get.sh

```diff
@@ -177,7 +177,29 @@ getBinaryOpenjdk()
 		fi
 	fi
 
-	if [ "$CUSTOMIZED_SDK_URL" != "" ]; then
+	if [ "$SDK_RESOURCE" == "nightly" ] && [ "$CUSTOMIZED_SDK_URL" != "" ]; then
+		if [ "$USERNAME" != "" ] && [ "$PASSWORD" != "" ]; then
+			curl_options="--user $USERNAME:$PASSWORD"
+		fi
```

Comment:
Please move the username/password outside of the if, so both line 180 and line 202 can use it.